### ISSUE TITLE
Badges not accesible in development workspace

### ISSUE THREAD

**Issue by lolodurante** [vtex-apps/badges#36]
**Describe the bug**
When I'm working in my development workspace and I create and save a badge, it does not appear on "Edit badges". Only when I am in master and go to the badges configuration i see the saved badges. This means the badges are only accesible in master and not in the development workspace.

**To Reproduce**

1. Go to 'admin/badges' in development workspace
2. Create a badge
3. In edit badges you won't see the badge you've just  created
4. If you go to master you'll see all the badges 

**Expected behavior**
I should be able to access all the badges in my development workspace for development utilizing the app. 

**Video**
[screen-capture (3).webm](https://user-images.githubusercontent.com/59238339/182860107-01fe6508-d3d0-470d-8bf4-9099d0c02ece.webm)

**Additional context**
Workspace to test : [vtex--beautycounterqa.myvtex.com/admin/badges
In the store framework we used the same code as the documentation here : https://developers.vtex.com/vtex-developer-docs/docs/vtex-badges

**Comment by gabrielzig** [comment]
I don't know if you managed to solve it, I had the same problem, but here's the resolution if someone faces the same problem.

In this case I'm using the name workspace **dev**, change it to yours.

1. Manually create a schema for the workspace using the Masterdata V2 API (Save schema by name API) where you can copy an existing schema (Get schema by name APIs). 

You can copy this schema:
```
curl --location --request GET 'https://{{accountName}}.{{environment}}.com.br/api/dataentities/vtex_badges_badges/schemas/3.5.0
```

2. Using the dev name workspace as an example, the schema name would be 3.5.0-**dev**, where at the beginning is the version of the app vtex.bagdes and then the name of the development workspace.
The name of the entity used in this app is: **_vtex_badges_badges_**.
```

curl --location --request PUT 'https://{{accountName}}.{{environment}}.com.br/api/dataentities/vtex_badges_badges/schemas/3.5.0-dev

-data-raw '{
    "$schema": "http://json-schema.org/draft-07/schema#",
    "title": "Badges",
    "type": "object",
    "properties": {
        "content": {
            "type": "string"
        },
        "name": {
            "type": "string"
        },
        "operator": {
            "type": "string"
        },
        "priority": {
            "type": "number"
        },
        "type": {
            "type": "string"
        },
        "simpleStatements": {
            "type": "array",
            "items": {
                "type": "object",
                "properties": {
                    "object": {
                        "type": "object",
                        "properties": {
                            "id": {
                                "type": "string"
                            },
                            "name": {
                                "type": "string"
                            },
                            "value": {
                                "type": "string"
                            }
                        },
                        "required": []
                    },
                    "subject": {
                        "type": "string"
                    },
                    "verb": {
                        "type": "string"
                    }
                },
                "required": [
                    "object",
                    "subject",
                    "verb"
                ]
            }
        }
    },
    "required": [
        "content",
        "name",
        "operator",
        "type",
        "simpleStatements",
        "priority"
    ],
    "v-indexed": [
        "content",
        "name",
        "operator",
        "type",
        "simpleStatements",
        "priority"
    ],
    "v-triggers": []
}'
```

After that, just wait for the indexing and the badges should appear in the development environment without any problems.

Hope this helps ;)

**Comment by christianAcctGlobal** [comment]
@gabrielzig , thank you very much, your solution worked for us!!!

we changed the version from `3.5.0` to `3.10.0` and it worked!

**Comment by lolodurante** [comment]
Yup. The schema is not created on workspaces other than master. It sucks that for every new workspace we have to do this but at least there's a solution. Thanks

**Comment by gaucho2512** [comment]
Hola buenas tardes , tengo un problema con las insignias , me pasaba lo mismo con el tema del workspace , pero eso ya lo solucione , use el mismo schema del ejemplo y  ahora en DEV si puedo editar mis etiquetas! , pero en el front no son visibleles.
![image](https://user-images.githubusercontent.com/67393137/212767170-980336c6-e97b-4263-bf07-42f376eb9eea.png)

Tampoco me busca todas las categorias segun las condiciones! , tengo muchisimas colecciones alrededor de 3000 pero solo me aparecen muy pocas.
![image](https://user-images.githubusercontent.com/67393137/212767547-9595a9af-7c04-4620-9679-e0f8e704bcf0.png)

En el front el bloque aparece , pero sin contenido
![image](https://user-images.githubusercontent.com/67393137/212767987-9849a0e1-0717-407a-82db-7e18a9815846.png)

ALguien me puede ayudar de porque pasa esto? , y si la app realmente funciona?
